atc/db/resource_config_scope.go

```diff
@@ -196,7 +214,7 @@ func (r *resourceConfigScope) UpdateLastCheckStartTime() (bool, error) {
 
 	updated, err := checkIfRowsUpdated(tx, `
 		UPDATE resource_config_scopes
-		SET last_check_start_time = now()
+		SET last_check_start_time = now(), last_check_succeeded = false
```

Comment:
is it necessary to set it to `false` here? seems like it'd be safe to leave it as the previous value since it won't be checked until the lock is released (after `UpdateLastCheckEndTime`)

i could see other things using this column in the future to indicate checking status (besides the initial check - i guess technically we could allow `NULL` for that), so it may be noisy to have it flip-flop to 'false' all the time. not a huge deal though - just wondering if this is actually needed and i missed something.

Reply:
Make sense.